### PR TITLE
Styles buttons all across our layouts

### PR DESCRIPTION
**Pull request by Samgalgo**
## Description
This PR styles all the buttons across our layouts.

## Background Information
In a previous PR I made some code changes to our **Delete** button so it could work properly, by making that change the style of the button changed so I needed to restyle all my other buttons.

## Changelog

Internal changes
-Added style and a few empty <div><h1></h1></div> to create space at the bottom of the layouts for the footer would not cover them. Not a real fix but looking into fixing that part without empty code.

External changes
-Sign in, sign up buttons changed color.
-The footer is no longer on top of the buttons
-The position on the buttons show, edit changed a bit.

## Implications for `dev` branch

No

## Screenshots/Recordings

(Please include before/after screenshots of any user-facing changes)

## PR Checklist

- [ ] Write clear, easy to read description of PR
- [ ] Remove all text from PR description that is in (parenthesis)
- [ ] Add and label screenshots (if dealing with UI. if not, delete this line)
- [ ] Remove debugging statements like `console.log`, `byebug`, `puts`, `prints`, `p`, `pp`

## How does this PR make you feel?

Good a little troubled that I did a cheap fixed so the footer doesn't cover the buttons.

### CODE REVIEW

File: app/views/groups/_update.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<%= form_with(model: group) do |form| %>
```
Comment (myang1010) [inline app/views/groups/_update.html.erb]: This file is mostly the same as `_form`. Seems you want to change the button name?
If so, this[ thread](https://stackoverflow.com/questions/35815810/suplying-different-names-for-submit-buttons-in-forms) may help.

File: app/views/groups/edit.html.erb [hunk 2]
```diff
@@ -3,9 +3,8 @@
         <div class="col-4 text center">
             <h1>Editing Group</h1>
 
-            <%= render 'form', group: @group %>
+            <%= render 'update', group: @group %>
```
Comment (myang1010) [inline app/views/groups/edit.html.erb]: If you remove the `_update` partial, don't forget the change here.

File: app/views/groups/index.html.erb [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
         </div>
       </div>
       <br>
     <% end %>
+      
   <br>
    
       <%= link_to 'New Group', new_group_path, class: "btn-create" %>
     
+    <div>
```
Comment (myang1010) [inline app/views/groups/index.html.erb]: This is not suggested. The `footer` could be change in a second. Then you have to come back and change all the files.

**Review by myang1010 — CHANGES_REQUESTED** [review]
Thanks for the effort for styling the buttons. Suggest a few changes below.

File: app/views/groups/_update.html.erb (hunk 1 above)
Comment (Samgalgo) [reply app/views/groups/_update.html.erb]: Thank you Ming, when I added the _update partial I immediately thought about the DRY principle haha, and had the feeling there had to be a different way to display the name of the buttons. Thank you for suggesting this article!

**Review by myang1010 — APPROVED** [review]

**Pull request merged**